Header header
float32[] data
geometry_msgs/Pose pose
# pose of the (0,0,0) and the coordinate system
geometry_msgs/Vector3 resols
uint32 size_x
uint32 size_y
uint32 size_z
